app/presenters/notifications/index.presenter.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+/**
+ * Formats data for the `/notifications` page
+ * @module NotificationsIndexPresenter
```

Comment:
Regarding the name of this file `index.presenter.js` I think there was a time when we were using that filename in our services/presenters etc but I don't thinnk we do anymore. I did a search in this repo for files named `index.` and can't find any except for the one in the root. It would appear the ones that were named index have since been renamed, for example `app/presenters/bill-runs/index-bill-runs.presenter.js`. So if we are following that pattern I think this file should be renamed to `index-notifications.presenter.js`